script/Arcueid/Day 7/07_00_ARC07_3_1_1.txt

```diff
@@ -151,38 +152,40 @@ From here on, we're dealing with creatures that could arguably be referred to as
 [sha:ded0f97af52d5348b2e55a68eaf848bdaa7e7bc6]{
 -- Page 529, Offset 17887.
 -- 生きる屍。アンデッド。生前ほどの思考能力はないものの、自分だけで人間生活を偽装できる。
-A living corpse. Undead.
+Living corpses, the Undead.
  Their cognitive abilities aren't what they once were,
- but they can still blend in with human society by themselves.
+ but they can still blend in with human society on their own.
 }
 [sha:55062a4088c7fa70c1b281c1bf1f811dfc293c4a]{
 -- Page 529, Offset 17888.
 -- 一度死んでから脳を再構成した上級兵士。日光も平気よ。ただし防腐処理はこまめにしないと、あっさり正体が露見する。
 These are higher-ranking soldiers,
  whose brains have been completely reconstructed since their initial death.
  Sunlight doesn't faze them either.
- However, their true form can easily be exposed if they don't take proper measures to prevent decay.
+ However, their true form can easily be exposed if they don't perform embalming on themselves.
 }
 [sha:a685cec6f7cbcd08cf3d0ab241ce4dd92d6c6f9f]{
 -- Page 529, Offset 17889.
 -- 死者と呼ばれるカテゴリーはここまでね」
-This is the last of what we consider the Dead."
+These are all of what we call the Dead."
 }
 [sha:5599af88891f6a10c967bbece2e2a2b0af4a0991]{
 -- Page 530, Offset 17890.
 -- ……なるほど。
-...I see.
+I see...
 }
 [sha:7df695cf62f6207ecfc7682eaebc03e3d290fac0]{
 -- Page 530, Offset 17891.
 -- あのゾンビ……死者にもそんな段階があったのか。
-That zombie... So even the Dead are split into different classes.
+I guess there are tiers even to those zombies... these 'Dead'.
```

Comment:
It was intentional actually!

"Those zombies" as in things Shiki have been seeing (more distant)
"These dead" as in what they were just talking about (closer)

That said, it might be a bit too irregular for some, but know that speech is more powerful than regularity.
If you insist I can change it, but I actually think it got the point across better.